app/core/configs.py

```diff
@@ -66,31 +54,8 @@ class RootEdgeConfig(BaseModel):
     Root configuration for edge inference.
     """
 
-    local_inference_templates: Dict[str, LocalInferenceConfig]
-    detectors: Dict[str, DetectorConfig]
-
-    @model_validator(mode="after")
-    def validate_templates(self):
-        """
-        Validate the templates referenced by the detectors.
-        :param values: The values passed to the validator. This is a dictionary of the form:
-            {
-                'detectors': {
-                    'detector_1': DetectorConfig(
-                                    detector_id='det_123',
-                                    local_inference_template='default',
-                                    always_return_edge_prediction=False
-                                )
-                },
-                'local_inference_templates': {
-                    'default': LocalInferenceConfig(
-                                    enabled=True,
-                                    refresh_rate=120.0
-                                )
-                }
-            }
-        """
-        for detector in self.detectors.values():
-            if detector.local_inference_template not in self.local_inference_templates:
-                raise ValueError(f"Local Inference Template {detector.local_inference_template} not defined.")
-        return self
+    refresh_rate: float = Field(
```

Comment:
This is now explicitly a global parameter (it actually always was, but it pretended to be detector-specific).